From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kyr Shatskyy <kyrylo.shatskyy@clyso.com>
Date: Tue, 23 Jul 2024 19:40:23 +0200
Subject: [PATCH] mgr/osd_perf_query: fix ivalid escape sequence

Get rid of warning which, according [1], is going to become an error:

osd_perf_query/module.py:61
  /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/osd_perf_query/module.py:61: DeprecationWarning: invalid escape sequence '\.'
    'regex': '^(?:rbd|journal)_data\.(?:([0-9]+)\.)?([^.]+)\.'},

1. https://docs.python.org/3/library/re.html

Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@clyso.com>
---
 src/pybind/mgr/osd_perf_query/module.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pybind/mgr/osd_perf_query/module.py b/src/pybind/mgr/osd_perf_query/module.py
index 6f87c1d907a..ac32f804282 100644
--- a/src/pybind/mgr/osd_perf_query/module.py
+++ b/src/pybind/mgr/osd_perf_query/module.py
@@ -58,7 +58,7 @@ class OSDPerfQuery(MgrModule):
         'key_descriptor': [
             {'type': 'pool_id', 'regex': '^(.+)$'},
             {'type': 'object_name',
-             'regex': '^(?:rbd|journal)_data\.(?:([0-9]+)\.)?([^.]+)\.'},
+             'regex': r'^(?:rbd|journal)_data\.(?:([0-9]+)\.)?([^.]+)\.'},
         ],
         'performance_counter_descriptors': [
             'bytes', 'write_ops', 'read_ops', 'write_bytes', 'read_bytes',
